apps/condo/domains/property/schema/Property.js

```diff
@@ -38,6 +40,18 @@ const Property = new GQLListSchema('Property', {
             schemaDoc: 'Normalized address',
             type: Text,
             isRequired: true,
+            hooks: {
+                validateInput: async ({ resolvedData, fieldPath, addFieldValidationError, context, existingItem }) => {
+                    const value = resolvedData[fieldPath]
+                    // if we create property or we update property address
```

Comment:
seems like it's better to watch on "operation" value and match this one with 'create' or 'update' value

Reply:
made a check operation instead existingItem in ad47814